plugins/ulimit-adjuster/README.md

```diff
@@ -0,0 +1,36 @@
+## ulimit Adjuster Plugin
+
+This sample plugin can adjust ulimits for containers using pod annotations.
+
+### Annotations
+
+ulimits are annotated using the key
+`ulimits.nri.containerd.io/container.$CONTAINER_NAME`, which adjusts ulimits
+for `$CONTAINER_NAME`.  The ulimit names are the valid names of Linux resource
+limits, which can be seen on the
+[`setrlimit(2)` manual page](https://linux.die.net/man/2/setrlimit).
+
+The annotation syntax for ulimit adjustment is
+
+```
+- type: RLIMIT_NOFILE
+  soft: 1024
+  hard: 4096
+- path: RLIMIT_MEMLOCK
+  soft: 1073741824
+  hard: 1073741824
+  ...
+```
+
+All fields are mandatory (`soft` and `hard` will be interpreted as 0 if
```

Comment:
is there relationship between those? Should hard default to soft if not defined?

Reply:
Hard should be >= soft.  I can add a validation for that.

Reply:
Validation added in 5ecea04.